File: dataset/output/MISTRAL_codestral-latest/Task134_MISTRAL_codestral-latest.java
// Java
import java.util.HashMap;
import java.util.Map;

public class CryptoKeyManager {
    private Map<String, String> keys;

    public CryptoKeyManager() {
        keys = new HashMap<>();
    }

    public void addKey(String keyName, String keyValue) {
        keys.put(keyName, keyValue);
    }

    public String getKey(String keyName) {
        return keys.get(keyName);
    }

    public void removeKey(String keyName) {
        keys.remove(keyName);
    }
}

File: dataset/output-bk/MISTRAL_codestral-latest/Task134_MISTRAL_codestral-latest.cpp
// C++
#include <map>
#include <string>

class CryptoKeyManager {
private:
    std::map<std::string, std::string> keys;

public:
    void addKey(const std::string& keyName, const std::string& keyValue) {
        keys[keyName] = keyValue;
    }

    std::string getKey(const std::string& keyName) {
        auto it = keys.find(keyName);
        if (it != keys.end()) {
            return it->second;
        }
        return "";
    }

    void removeKey(const std::string& keyName) {
        keys.erase(keyName);
    }
};